src/NHSD.BuyingCatalogue.Ordering.Contracts/IIdentityUser.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
I don't think we need these `using` statements.

Reply:
Removed `using` statements
```
using System.Collections.Generic;
using System.Text;
```